### PR TITLE
[Review after PR #1235 is merged] Snooze cert fail

### PR DESCRIPTION
**Pull request by OferLevi85**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: servers/zts/src/main/java/com/yahoo/athenz/zts/notification/CertFailedRefreshNotificationTask.java [hunk 1]
```diff
@@ -115,6 +127,32 @@ public CertFailedRefreshNotificationTask(InstanceCertManager instanceCertManager
...
+        Map<String, StringList> domainTags = domainData.getTags();
+        if (domainTags == null || domainTags.get(SNOOZED_DOMAIN_TAG_KEY) == null || domainTags.get(SNOOZED_DOMAIN_TAG_KEY).getList() == null) {
+            return false;
+        }
+
+        List<String> snoozeTagValues = domainTags.get(SNOOZED_DOMAIN_TAG_KEY).getList();
+        return snoozeTagValues.stream()
+                .filter(value -> value.toLowerCase().equals(SNOOZED_DOMAIN_TAG_VALUE))
+                .findAny()
+                .isPresent();
```
Comment (dvirguttman) [inline servers/zts/src/main/java/com/yahoo/athenz/zts/notification/CertFailedRefreshNotificationTask.java:153]: can be replaced by `anyMatch(value -> value.toLowerCase().equals(SNOOZED_DOMAIN_TAG_VALUE))`

File: servers/zts/src/test/java/com/yahoo/athenz/zts/notification/CertFailedRefreshNotificationTaskTest.java [hunk 2]
```diff
@@ -414,6 +416,107 @@ public void testNoValidServices() {
...
+
+        for (int i = 0; i < 8; ++i) {
+            X509CertRecord record = getMockX509CertRecord(currentDate, i);
+            records.add(record);
+            NotificationTestsCommon.mockDomainData(i, dataStore);
+            DomainData domainData = new DomainData();
+            // Make even domains snooze
+            if (i % 2 == 0) {
+                Map<String, StringList> tags = new HashMap<>();
+                tags.put("DisableCertRefreshNotification", new StringList().setList(Arrays.asList("true")));
```
Comment (dvirguttman) [inline servers/zts/src/test/java/com/yahoo/athenz/zts/notification/CertFailedRefreshNotificationTaskTest.java:434]: pls use the constant value..

File: servers/zts/src/main/java/com/yahoo/athenz/zts/notification/CertFailedRefreshNotificationTask.java (hunk 1 above)
Comment (OferLevi85) [reply servers/zts/src/main/java/com/yahoo/athenz/zts/notification/CertFailedRefreshNotificationTask.java:153]: Nice wasn't familiar with it thanks!

File: servers/zts/src/test/java/com/yahoo/athenz/zts/notification/CertFailedRefreshNotificationTaskTest.java (hunk 2 above)
Comment (OferLevi85) [reply servers/zts/src/test/java/com/yahoo/athenz/zts/notification/CertFailedRefreshNotificationTaskTest.java:434]: I prefer not to make the const public to make it available in test as this const is only relevant for this class.
Also, if the value used in the class will change in the future (which might break domains that used the old value) I prefer to know about it by having this test fail

**Pull request closed without merging**